include/bactria/bactria.hpp

```diff
@@ -14,10 +14,47 @@
  */
 
 #ifndef BACTRIA_BACTRIA_HPP
-#define BACTRIA_BACTRIA_HPP
+#   define BACTRIA_BACTRIA_HPP
 
-#include <bactria/event.hpp>
-#include <bactria/phase.hpp>
-#include <bactria/region.hpp>
+#   include <bactria/Event.hpp>
+#   include <bactria/Plugin.hpp>
+#   include <bactria/Range.hpp>
+#   include <bactria/Phase.hpp>
+#   include <bactria/Sector.hpp>
+
+#   include <memory>
+
+namespace bactria
+{
+    struct Context
+    {
+        Context(plugin::plugin_handle_t h) noexcept : handle{h}
+        {}
+
+        Context(Context const&) noexcept = default;
+        auto operator=(Context const&) noexcept -> Context& = default;
```

Comment:
So I can copy around a `Context` as often as I want, but they all point to the same thing?

Reply:
Yes, multiple contexts are not required. One per process.

Reply:
There is no real downside to having multiple contexts at the present point, so I don't see a reason to forbid copying.